.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ references:
       - image: aeternity/infrastructure
     working_directory: ~/aeternity
     environment:
-      DOCKERHUB_REPO: aeternity/epoch
+      DOCKERHUB_REPO: aeternity/aeternity
```

Comment:
What is the plan for the following tags in aeternity/epoch?
* master
* minerva
* latest

I suggest:
* After this PR merged, delete `master` and `minerva` from aeternity/epoch.
* After aeternity/aeternity latest pushed as 1.3.0, delete latest from aeternity/epoch.

Reply:
Good point I thought `latest` has been migrated as part of https://www.pivotaltracker.com/story/show/163298160

I'll migrate it before merging this.

Then once this is merged I'll delete master and minerva from aeternity/epoch and latest once 1.3.0 is tagged, unfortunately that will break the BC, maybe we push 1.3.0 and `latest` in `aeternity/epoch` as well to give some gratis period?

Reply:
`latest` has been pushed to the new repo https://hub.docker.com/r/aeternity/aeternity/tags